examples/medplum-patient-intake-demo/src/pages/UploadDataPage.tsx

```diff
@@ -72,10 +66,130 @@ export function UploadDataPage(): JSX.Element {
   );
 }
 
-async function uploadData(medplum: MedplumClient, data: Bundle): Promise<void> {
-  const result = await medplum.executeBatch(data);
+async function uploadCoreData(medplum: MedplumClient): Promise<void> {
+  const batch = coreData as Bundle;
+
+  const result = await medplum.executeBatch(batch);
+
+  showNotification({
+    icon: <IconCircleCheck />,
+    title: 'Success',
+    message: 'Uploaded Business Statuses',
+  });
+
+  if (result.entry?.every((entry) => entry.response?.outcome && isOk(entry.response?.outcome))) {
+    await setTimeout(
+      () =>
+        showNotification({
+          icon: <IconCircleCheck />,
+          title: 'Success',
+          message: 'Uploaded Business Statuses',
```

Comment:
"Business Status" is not the right description here. We need to update this message